src/main/webapp/js/reducers/entity-reducer.js

```diff
@@ -107,11 +112,21 @@ export default (state = initialState, action) => {
       });
     case UPDATE_ENTITY:
       return update(state, {
-        [entityType]: { $set: { [payload.data.id]: payload.data } },
+        [entityType]: {
+          $set: _.keyBy(_.map(payload.data, elem => ({
```

Comment:
you should not use the keyBy here, it's single element, you should only use map on payload.data